Kconfig.zephyr

```diff
@@ -338,6 +338,84 @@ endmenu
 
 menu "Compiler Options"
 
+config MINIMUM_REQUIRED_STD_C99
+	bool
+	help
+	  Hidden option to select compiler support C99 standard or higher.
```

Comment:
would be better to name these `REQUIRES_STD_Cxx` to align with the existing symbols of the same category (e.g. `REQUIRES_FULL_LIBC`, `REQUIRES_FULL_LIBCPP`, `REQUIRES_FLOAT_PRINTF`).

Reply:
nicely spotted.